docs/API-specification.md

```diff
@@ -0,0 +1,160 @@
+<!--
```

Comment:
I think CRD-specification is more appropriate.

Reply:
+1，agree